src/pages/TeamSettings/Service-Accounts-Table.vue

```diff
@@ -0,0 +1,486 @@
+          'error',
+          'Something went wrong while trying to delete your API key. Please try again. If this error persists, please contact help@prefect.io.'
+        )
+      }
+    },
+    handleAlert(type, message) {
+      this.alertType = type
+      this.alertMessage = message
+      this.alertShow = true
+    },
+    resetNewKey() {
+      this.newKeyName = ''
+      this.newKeyScope = ''
+      this.newAPIKey = ''
+      this.expiresAt = null
+    },
+    async removeAccount(id) {
+      this.isRemovingUser = true
+
+      const res = await this.$apollo.mutate({
+        mutation: require('@/graphql/User/delete-service-account.gql'),
+        variables: { id }
+      })
+
+      if (res?.data?.delete_service_account?.success) {
+        this.$emit('successful-action', 'The service account has been removed.')
+        this.$apollo.queries.tenantUsers.refetch()
+      } else {
+        this.$emit(
+          'failed-action',
+          'Something went wrong while trying to remove this service account. Please try again.'
+        )
+      }
+
+      this.isRemovingUser = false
+      this.dialogRemoveUser = false
+      this.selectedUser = null
+    }
+  },
+  apollo: {
+    tenantUsers: {
+      query: require('@/graphql/Tenant/tenant-users.gql'),
+      watchLoading(isLoading) {
+        this.isFetchingMembers = isLoading
+      },
+      result({ data }) {
+        if (!data) return
+
+        this.membersItems = data.tenantUsers
+          .filter(user =>
+            user.memberships.find(mem => mem.tenant_id == this.tenant.id)
+          )
+          .filter(user => user.account_type === 'SERVICE')
+          .map(user => {
+            user.memberships.find(mem => mem.tenant_id == this.tenant.id)
+
+            return {
+              id: user.id,
+              firstName: user.first_name
+            }
+          })
+        return data
+      },
+      error() {
+        this.$emit(
+          'failed-action',
+          'Something went wrong while trying to fetch your service accounts. Please try again.'
+        )
+        this.$emit('load-end')
+      },
+      fetchPolicy: 'no-cache',
+      skip() {
+        return !this.tenant
+      }
+    },
+    keys: {
+      query: require('@/graphql/Tokens/api-keys.gql'),
+      fetchPolicy: 'network-only',
+      error() {
+        this.handleAlert(
+          'error',
+          'Something went wrong while trying to fetch your API keys. Please refresh the page and try again. If this error persists, please email help@prefect.io.'
+        )
+      },
+      result({ data }) {
+        this.keys = data.auth_api_key.map(key => {
+          return {
+            id: key.id,
+            name: key.name,
+            created_at: key.created,
+            expires: key.expires_at,
+            user_id: key.user_id
+          }
+        })
+        this.isFetchingKeys = false
+      },
+      update: data => data
+    }
+  }
+}
+</script>
+
+<template>
+  <div>
+    <v-data-table
```

Comment:
Can we add a loading state?  I'm getting a 'this team does not have any service accounts' message flash up before the table loads. 
<img width="1107" alt="Screen Shot 2021-03-25 at 10 48 01 AM" src="https://user-images.githubusercontent.com/40272060/112492369-9667fa80-8d57-11eb-8859-75e3cbe79b32.png">